tcl/org.testeditor.tcl.model/src/org/testeditor/tcl/util/TclModelUtil.xtend

```diff
@@ -210,9 +210,12 @@ class TclModelUtil extends TslModelUtil {
 
 	def ValueSpaceAssignment getValueSpaceAssignment(Component component, TestStep container) {
 		for (element : component.elements) {
-			val valueSpace = getValueSpaceAssignment(element, container)
-			if (valueSpace != null) {
-				return valueSpace
+			val stepContentElementsStrings = container.contents.filter(StepContentElement).map[value]
```

Comment:
The model is not easy to understand. 
If we have more than value space in a component we need the test step variable call of the element name, to identify the correct valuespace assignment of the element.
for example:
execute menu item  "«value2»"  in tree <TestStepSelector>
we need to check the value TestStepSelector to identify from:
`			component ProjectExplorer is General {
				element ProjektBaum is TreeView {
					label = "Projekt Baum"
					locator ="Project Explorer"
					executeContextMenuEntry.item restrict to projectmenues 
				}
				element TestStepSelector is TreeView {
					label = "Teststep selector"
					locator ="teststepSelector"
					executeContextMenuEntry.item restrict to components 
				}
			}
`
See Test getTCLWithTwoValueSpaces in TClValidatorTest